src/vm/eetwain.cpp

```diff
@@ -2919,6 +2919,22 @@ bool InstructionAlreadyExecuted(unsigned walkOffset, unsigned actualHaltOffset)
     return (walkOffset < actualHaltOffset);
 }
 
+// Skips past a "vzeroupper" instruction if one is present. Returns offset unchanged
+// otherwise.
+inline unsigned SKIP_VZEROUPPER_IF_PRESENT(PTR_CBYTE base, unsigned offset)
+{
+    LIMITED_METHOD_CONTRACT;
+
+    if (CheckInstrByte(base[offset], 0xC5) &&
+        CheckInstrByte(base[offset + 1], 0xF8) &&
+        CheckInstrByte(base[offset + 2], 0x77))
```

Comment:
It would be nice to define a constant for the instruction code like we do for other instructions around line 20 in this file to match handling of other instructions.